src/goe/filesystem/goe_azure.py

```diff
@@ -118,7 +118,9 @@ def __init__(
 
         logger.info("Client setup: GOEAzure")
 
-        super(GOEAzure, self).__init__(messages, dry_run=dry_run)
+        super(GOEAzure, self).__init__(
+            messages, dry_run=dry_run, do_not_connect=do_not_connect
```

Comment:
Are `dry_run` and `do_not_connect` linked in some way? I suspect the former is a user option and the latter is a unit test option, but just checking.

Reply:
Not really. `dry_run` is about running offload commands without `-x`.
`do_not_connect` is about unit testing without trying to make connections. If I were writing this code afresh now then `do_not_connect` would not exist, but, for now, I'm just carrying on down a path I created for myself several years ago.